python/src/MonteCarloExperiment_doc.i.in

```diff
@@ -12,12 +12,12 @@ distribution : :class:`~openturns.Distribution`
     Distribution :math:`\mu` with an independent copula used to generate the
     set of input data.
 size : positive int
-    Number :math:`cardI` of points that will be generated in the experiment.
+    Number :math:`\sampleSize` of points that will be generated in the experiment.
 
 Notes
 -----
-MonteCarloExperiment is a random weighted design of experiments ([hammersley1961]_ page 51, [lemieux2009]_ page 3).
-The :meth:`generate` method computes the nodes :math:`(\inputReal_i)_{i = 1, ..., \sampleSize}`
+MonteCarloExperiment is a random weighted design of experiments (see [hammersley1961]_ page 51, [lemieux2009]_ page 3).
+The :meth:`~openturns.MonteCarloExperiment.generate` method computes the nodes :math:`(\inputReal_i)_{i = 1, ..., \sampleSize}`
```

Comment:
:meth:`generate` was fine, does it make a difference ?

Reply:
Fixed.